### PR TITLE
Only specify args in etcd-druid

### PR DESCRIPTION
**Pull request by maboehm**
**How to categorize this PR?**
/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Part of building our etcd-druid fork in prow with ko: https://dev.azure.com/schwarzit/schwarzit.ske/_workitems/edit/632172

**Special notes for your reviewer**:
Will also contribute this upstream and update the description once thats done.

Changing the version file is no longer needed with the build process in prow, see https://github.com/stackitcloud/ske-ci-infra/blob/main/config/jobs/gardener/gardener-images.yaml#L33-L41

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.88.3-ske-2
+v1.88.3
```
Comment (nschad) [inline VERSION:1]: Entfernen wir das jetzt einfach weil eh egal? (Bauen ja nicht mehr über concourse)?

**Review by nschad — COMMENTED** [review]
/lgtm

File: VERSION (hunk 1 above)
Comment (maboehm) [reply VERSION:1]: genau, hab ich in der PR description auch nochmal explizit erwähnt.

File: pkg/component/etcd/bootstrap.go [hunk 2]
```diff
@@ -386,9 +386,8 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
-func getDruidDeployCommands(etcdConfig *config.ETCDConfig) []string {
+func getDruidDeployArgs(etcdConfig *config.ETCDConfig) []string {
 	command := []string{
```
Comment (nschad) [inline pkg/component/etcd/bootstrap.go:390]: `command` would rename that too to `args`

Comment (maboehm) [reply pkg/component/etcd/bootstrap.go:390]: dang

**Pull request merged**